test/OpenTelemetry.Exporter.Geneva.Tests/OtlpProtobufMetricExporterTests.cs

```diff
@@ -187,6 +292,49 @@ public void CounterSerializationSingleMetricPoint(string instrumentName, long? l
             Assert.Equal(doubleValue, dataPoint.AsDouble);
         }
 
+#if EXPOSE_EXPERIMENTAL_FEATURES
+        if (isExemplarsEnabled)
+        {
+            Assert.Single(dataPoint.Exemplars);
+
+            var exemplar = dataPoint.Exemplars[0];
+            if (longValue != null)
+            {
+                Assert.Equal(longValue.Value, exemplar.AsInt);
+            }
+            else
+            {
+                Assert.Equal(doubleValue.Value, exemplar.AsDouble);
+            }
+
+            if (isTracingEnabled)
+            {
+                var spanIdBytes = new byte[8];
+                activity.SpanId.CopyTo(spanIdBytes);
+
+                for (int i = 0; i < 8; i++)
```

Comment:
nit: Use `Enumerable.SequenceEqual` to check if the two arrays have the same values.